另外，Github 的基于你的源代码的构建（npm run build）失败了，这里有写，但是我没明白是为什么。

https://github.com/MCSManager/UI/actions/runs/4977631724/jobs/8906910638?pr=97

```

This dependency was not found:

 ERROR  Build failed with errors.
* /home/runner/work/UI/UI in multi .

To install it, you can run: npm install --save /home/runner/work/UI/UI
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mcsmanager-ui@3.2.0 build: `vue-cli-service build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mcsmanager-ui@3.2.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2023-05-15T07_23_03_079Z-debug.log
Error: Process completed with exit code 1.
```

这个可能你需要看一下是因为什么而导致的，是否有错误的配置？

我猜是这里导致的：

https://github.com/MCSManager/UI/pull/97/files#diff-1d62a367f02f5c91000e35c82b0086fae3246eca3942f9034c058b16a3780ec2R15

--- a/vue.config.js
+++ b/vue.config.js
@@ -11,8 +11,8 @@
  */
 module.exports = {
   // options...
+  publicPath: "./",
   pages: {
-    publicPath: "./",
     index: {
       entry: "src/app/index.js",
       template: "public/index.html",

